Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.14-alpine
+
+RUN \
```

Comment:
We can separate env setup, git clone, and go build operations with multiple `RUN`.